ocgis/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+    name: ocgis
+    version: 1.2.0
+
+source:
+    git_url: https://github.com/NCPP/ocgis.git
+    git_tag: v1.2.0
+
+build:
+    number: 0
+    skip: True # [win]
+    skip: True # [osx]
+    skip: True # [py3k]
+
+requirements:
+    build:
+        - python >=2.7,<3
+        - cfunits-python
+        - fiona
+        - gdal <2
+        - libgdal <2
+        - netcdf4 <=1.1.6
```

Comment:
There is an odd issue with netCDF4-python v1.1.6+ that required this (very specific) workaround: https://github.com/NCPP/ocgis/blob/176a86e293e101877de1098a35f9efde73e83718/src/ocgis/interface/base/attributes.py#L32. I had trouble isolating the issue, but it seems like there is a rogue pointer somewhere in the netCDF4 attribute code. The workaround came after the 1.2 release, and it is fixed in the upcoming release. If this is causing a problem, I could add a patch file for the interim?